### ISSUE TITLE
Deb packages for bolt not available in aarch64

### ISSUE THREAD

**Issue by teeks99** [llvm/llvm-project#62695]
I am not seeing the libbolt-XX-dev and bolt-XX packages for arm64/aarch64, they are available for the x86_64/amd64 architecture.

**Comment by teeks99** [comment]
@sylvestre I think this will be assigned to you. Needs the `packaging` label.

**Comment by sylvestre** [comment]
yeah, because libbolt_rt_hugify.a wasn't generated on arm64. is that fixed?

**Comment by ilinpv** [comment]
@sylvestre Hugify support for BOLT on AArch64 has landed in commit 38faf32d23d1edb7708711d0f9e0db853ddf75a5 and runtime libraries should now be generated. Are there any remaining issues or blockers preventing you from building deb packages for bolt?
